tjpcov/covariance_builder.py

```diff
@@ -297,15 +297,20 @@ def get_covariance_block_for_sacc(
 
         # Let's use only one of the data types since all of them will have to
         # be e.g. cl, xi for the same tracer combination
-        dtypes1 = self.get_tracer_comb_data_types(tracer_comb1)[0]
-        dtypes2 = self.get_tracer_comb_data_types(tracer_comb2)[0]
+        dtypes1 = self.get_tracer_comb_data_types(tracer_comb1)
+        dtypes2 = self.get_tracer_comb_data_types(tracer_comb2)
 
         s = self.io.get_sacc_file()
 
         # Check in the given order
-        correct_dtypes = (self._tracer_types[0] in dtypes1) and (
-            self._tracer_types[1] in dtypes2
+        correct_dtype1 = any(
```

Comment:
Before, we were only taking the first data type in our sacc file.  For example, if I wanted to only compute cluster mass, it would always fail, because cluster counts also exist for the same tracer. 

Now we check to ensure that the data type I want to calculate my covariance for exists in the list of data types.  

Your scenario is possible and will happen, but later when you place the blocks in their place if you _dont_ specify data type.